doc/devel/gitwash/matplotlib_for_dev.rst

```diff
@@ -0,0 +1,20 @@
+.. _matplotlib-for-dev:
+
+=================================================
+Installing matplotlib from source for development
+=================================================
+
+After obtaining a local copy of the matpotlib source code (:ref:`set-up-fork`),
+navigate to the matplotlib directory and run the following in the shell:
+
+::
+    
+    python setup.py develop
```

Comment:
I agree that we should have another mean of setting up a development environment. Unfortunately, because matplotlib's code organization is very un standard, it is hard to do.
I used to reinstall matplotlib every time, but it has become a pain now that the git hash is appended to the version.